### PR TITLE
Update Airbnb style and added Nexton best practice 

### PR DESCRIPTION
**Pull request by Salvador-ON**
## Background
Update Nexton Code Style with latest updates
## Changes done
* Creation of Nexton Best Practice readme. that include the best practice to do on React and encourage the use of Hooks instead of classes.
* Update Eslint Airbnb style code removing obsolete information, adding deprecations warnings, and Hooks rules.
## Pending to be done
* Create a third document related to styled-components
## Notes
After some research, the actual config using eslint + prettier is the most completed solution for react linting
we could make our listing stronger adding other plugins like eslint-plugin-jsx-a11y to improve accessibility

### CODE REVIEW

File: React/Readme.md [hunk 1]
```diff
@@ -20,8 +21,6 @@ Forked from the excellent [Airbnb React Style Guide](https://github.com/airbnb/j
   1. [Refs](#refs)
   1. [Parentheses](#parentheses)
   1. [Tags](#tags)
-  1. [Methods](#methods)
-  1. [Ordering](#ordering)
   1. [`isMounted`](#ismounted)
```
Comment (kilisoria) [inline React/Readme.md:24]: Maybe We could remove it directly due it's an old feature and it's close to will be deprecated.

File: React/Readme.md [hunk 2]
```diff
@@ -32,60 +31,37 @@ Forked from the excellent [Airbnb React Style Guide](https://github.com/airbnb/j
   - Do not use `React.createElement` unless you’re initializing the app from a file that is not JSX.
   - [`react/forbid-prop-types`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/forbid-prop-types.md) will allow `arrays` and `objects` only if it is explicitly noted what `array` and `object` contains, using `arrayOf`, `objectOf`, or `shape`.
```
Comment (kilisoria) [inline React/Readme.md:33]: We could add statement to add `PropTypes` as un rule in a component. What do you think?

File: React/Readme.md [hunk 3]
```diff
@@ -32,60 +31,37 @@ Forked from the excellent [Airbnb React Style Guide](https://github.com/airbnb/j
...
-
-    // good
-    function Listing({ hello }) {
-      return <div>{hello}</div>;
-    }
-    ```
+  ### Only Call Hooks from React Functions
+  - Don’t call Hooks from regular JavaScript functions. Instead, you can:
+  - Call Hooks from React function components.
+  - Call Hooks from custom Hooks
```
Comment (kilisoria) [inline React/Readme.md:43]: Great! 👍  
Maybe We could add code examples regarding good and bad practices about it. What do you think?

File: React/Readme.md [hunk 4]
```diff
@@ -32,60 +31,37 @@ Forked from the excellent [Airbnb React Style Guide](https://github.com/airbnb/j
...
 
 ## Mixins
```
Comment (kilisoria) [inline React/Readme.md:45]: We could remove it directly due it's not a practice used. Maybe We could put in the section depractions, mention to it and `IsMount` What do you think?

File: React/Nexton Best Practice/Readme.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
+# React.JS, Nexton Best Practice.
```
Comment (kilisoria) [inline React/Nexton Best Practice/Readme.md:1]: I like It, awesome! 👍 
Maybe I think We should move to the `Readme` directly, We could put it to bottom then We have all information in one place. What do you think?

File: React/Nexton Best Practice/Readme.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+- The code we write should behave as expected, and be testable easily and quickly. It’s a good practice to name your test files identical to the source files with a ```.test``` suffix. It’ll then be easy to find the test files.
+
+## File Structure
+
+- We organize all the files by function type ```components```, ```actions```, ```services```.
+
+
+### Components Structure
+- If there’s any small component used by a particular component only, it makes sense to keep these smaller components all together within that component folder. The hierarchy will then be easy to understand – large components have their own folder and all their smaller parts are split into sub-folders. This way, we can easily extract code to any other project or even modify the code whenever we want.
```
Comment (kilisoria) [inline React/Nexton Best Practice/Readme.md:73]: Good point, it's a topic project Structure :) Maybe We could follow the approach We're using in the React Starter V2, Could you look at it please? Let me know your thoughts. Thanks.

File: React/Readme.md (hunk 4 above)
Comment (Salvador-ON) [reply React/Readme.md:45]: Sure !

File: React/Nexton Best Practice/Readme.md (hunk 5 above)
Comment (Salvador-ON) [reply React/Nexton Best Practice/Readme.md:1]: Yeah that could be great

File: React/Readme.md (hunk 2 above)
Comment (Salvador-ON) [reply React/Readme.md:33]: Sure I'll add it

File: React/Readme.md (hunk 3 above)
Comment (Salvador-ON) [reply React/Readme.md:43]: sure

File: React/Nexton Best Practice/Readme.md (hunk 6 above)
Comment (Salvador-ON) [reply React/Nexton Best Practice/Readme.md:73]: Ok I will check it

**Pull request merged**